api/zulip/__init__.py

```diff
@@ -230,32 +241,33 @@ def __init__(self, email=None, api_key=None, config_file=None,
         self.client_name = client
 
         if insecure:
-            self.tls_verification = False
+            self.tls_verification=False
```

Comment:
You're failing a linter test here:
`api/zulip/__init__.py:244:34: E225 missing whitespace around operator`

Could you fix it?